include/arbor/arbexcept.hpp

```diff
@@ -95,4 +95,11 @@ struct range_check_failure: arbor_exception {
     double value;
 };
 
+// Python wrapper errors
```

Comment:
In order to use `pprintf` from `arbor/util/strprintf.hpp` I would need to set the include path in cmake to `arbor/util`, otherwise the compiler won't find `util/meta.hpp` which gets included in `arbor/util/strprintf.hpp`. Or have duplicates or not use `pprintf` or leave python exception where it is. Any suggestions?

Reply:
Done.